nipoppy_cli/docs/source/conf.py

```diff
@@ -31,13 +31,20 @@
 
 nitpicky = True
 
-
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
 html_theme = "furo"
 html_static_path = ["_static"]
 
+# -- Furo configuration ------------------------------------------------------
+#  https://pradyunsg.me/furo/customisation/#customisation
+html_theme_options = {
+    "source_repository": "https://github.com/neurodatascience/nipoppy",
+    "source_branch": "main",
+    "source_directory": "docs/",
```

Comment:
Wrong docs directory
```suggestion
    "source_directory": "nipoppy_cli/docs/",
```
Though I guess we could move it outside of `nipoppy_cli` since I removed the old `docs` directory (which only had a README), but that will require a number of path changes in the docs and the root README